### PR TITLE
finish suggested video

### PR DESCRIPTION
**Pull request by ZoeCN**

### CODE REVIEW

File: src/client/components/SuggestedVideos/Suggested-Videos-View/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const SuggestedVidesView = (props) => (
+  <div className="suggested-video-container">
```
Comment (ikarasz) [inline src/client/components/SuggestedVideos/Suggested-Videos-View/index.js]: this is typically an aside

File: src/client/components/SuggestedVideos/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import './index.scss';
+import noPreview from './404.png';
+import SuggesstedVideosView from './Suggested-Videos-View';
```
Comment (ikarasz) [inline src/client/components/SuggestedVideos/index.js]: check this typo please and the folder name conventions

File: src/client/components/SuggestedVideos/Suggested-Videos-View/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <ul className="suggested-video-ul">
+      <h1>Up next</h1>
+      <li className="suggested-video-list">
```
Comment (ikarasz) [inline src/client/components/SuggestedVideos/Suggested-Videos-View/index.js]: remove the test items before merge

File: src/client/App.js [hunk 4]
```diff
@@ -2,13 +2,14 @@ import React from 'react';
...
 import Login from '../client/components/Login';
+import SuggestedVideos from '../client/components/SuggestedVideos';
 
 class App extends React.Component {
   render() {
     return (
       <Router>
         <div>
-          <Route exact path="/" component={Signup} />
+          <Route exact path="/" component={SuggestedVideos} />
```
Comment (ikarasz) [inline src/client/App.js]: remove test statement before merge

File: src/client/components/SuggestedVideos/index.scss [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+      border-radius: 100px;
+      bottom: .3rem;
+      color: $font-color;
+      font-size: .625rem;
+      height: .937rem;
+      line-height: .937rem;
+      position: absolute;
+      right: .5rem;
+      text-align: center;
+      width: 2.937rem;
```
Comment (ikarasz) [inline src/client/components/SuggestedVideos/index.scss]: you should set the padding instead

File: src/client/components/SuggestedVideos/index.scss [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+      $backround-color: #000;
+      background-color: rgba($color: $backround-color, $alpha: 0.73);
+      border-radius: 100px;
+      bottom: .3rem;
+      color: $font-color;
+      font-size: .625rem;
+      height: .937rem;
+      line-height: .937rem;
+      position: absolute;
+      right: .5rem;
```
Comment (ikarasz) [inline src/client/components/SuggestedVideos/index.scss]: you can use pixels here, and the same value should be set as for the bottom property

File: src/client/components/SuggestedVideos/Suggested-Videos-View/index.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+        <div className="suggested-video">
+          <a></a>
+          <span>1:00</span>
+        </div>
+        <p className="title">video test</p>
+        <span className="video-message">
+          <span>author-test</span>
+          <span>100 views</span>
```
Comment (ikarasz) [inline src/client/components/SuggestedVideos/Suggested-Videos-View/index.js]: in this case the user wont be navigated to the video if he/she clicks on the span tag

**Review by ikarasz — CHANGES_REQUESTED** [review]

**Comment by ZoeCN** [comment]
fixed

**Review by ikarasz — APPROVED** [review]
- suggestedVides
- put the ul into the list

**Pull request merged**